Training25/Program.cs

```diff
@@ -3,11 +3,50 @@
 // Copyright (c) Metamation India.
 // ------------------------------------------------------------------
 // Program.cs
-// Program on main branch.
+// Program to convert a given decimal number to different forms (Hexadecimal, Binary)
 // ------------------------------------------------------------------------------------------------
+using System.Text;
+
 namespace Training25;
+
 internal class Program {
    static void Main (string[] args) {
-      Console.WriteLine ("Hello, World!");
+      do {
+         Console.Write ("\nNumber Conversion Game\n~~~~~~~~~~~~~~~~~~~~~~\nInput : ");
+         if (int.TryParse (Console.ReadLine (), out int decValue)) {
+            string bin = Convert.ToString (decValue, 2), hex = decValue.ToString ("X");

```

Comment:
```suggestion
            string bin = decValue.ToString ("B"), hex = decValue.ToString ("X");
```
If you are using the numeric format use for both or Convert.ToString for both.